deploy/kcp/apiexport_jvmbuildservice.yaml

```diff
@@ -10,15 +10,17 @@ spec:
   - group: ""
     resource: "secrets"
   - group: ""
-    resource: "configmaps"
+    resource: "serviceaccounts"
+  - group: ""
+    resource: "services"
```

Comment:
`services` resource is not a kcp built-in type so you need to claim it via the identityHash referenced in kubernetes APIExport. Just add the same hash as in deployments and pipelineruns and also update the APIBinding accordingly and that probably should fix the get/create issue for services